### ISSUE TITLE
Surroundings helper does not capture every building

### ISSUE THREAD

**Issue by reyery** [architecture-building-systems/CityEnergyAnalyst#2815]
When the zone geometry bounds are of irregular shape as one like below, there is a huge chunk of buildings that are not considered as highlighted in red.

![image](https://user-images.githubusercontent.com/34395415/89852857-6140da80-dbc2-11ea-8809-2e60374ba311.png)

**Comment by martin-mosteiro** [comment]
I have the same issue here:
![Screen Shot 2020-08-20 at 13 10 01](https://user-images.githubusercontent.com/18548065/90763196-892cee00-e2e6-11ea-9984-9d1c86ffb561.png)

This is a duplicate of #2456 though – Which we closed because (a) no one was blocked by this issue, (b) we had no capacity to solve this issue, (c) we had a workaround. This *should* be solved eventually, though.

**Comment by daren-thomas** [comment]
@reyery good job! I think maybe you need to update the estimate on this one?

**Comment by reyery** [comment]
Closed by #2824
